client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -44,3 +44,21 @@ export const getStateData = ( country, state ) => {
 
 	return stateData;
 };
+
+/**
+ * Return a "sorted" list of countries, with a subset pulled to the top,
+ * and the rest sorted alphabetically.
+ * @param {array} list  List of countries to sort
+ * @return {array} sorted list of countries
+ */
+export function sortPopularCountriesToTop( list ) {
+	const popularCodes = [ 'AU', 'BR', 'CA', 'FR', 'DE', 'IT', 'ES', 'SE', 'GB', 'US' ];
+	const popularCountries = filter( list, item => -1 !== popularCodes.indexOf( item.code ) );
```

Comment:
I was going to see if we need to check for `list` to be an array first, but TIL `filter` will by default return an empty array if the input is undefined.